fixtures/k8s/kubernetes_resource_ingress_pass.yaml

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Can we add an example that creates a pod, and waits for it to terminate and then using test check that the terminate exit code is 0